OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectAtom.cs

```diff
@@ -220,6 +232,26 @@ sealed class DreamMetaObjectAtom : IDreamMetaObject {
                     return new DreamValue(VerbLists[dreamObject]);
                 case "filters":
                     return new DreamValue(_filterLists[dreamObject]);
+                case "color":
+                    if(!_atomManager.TryGetAppearance(dreamObject, out IconAppearance? appearance)) {
+                        //This is here to avoid infinite loops, since the lazy-init of appearances
+                        // (that would occur if we tried to just invoke MustGetAppearance())
+                        //requires accessing the color var.
+                        if (dreamObject.ObjectDefinition.TryGetVariable("color",out var colorValue)) {
+                            return colorValue;
+                        }
+                        return DreamValue.Null; //idek
+                    }
+                    if(appearance!.SillyColorFilter is not null) {
```

Comment:
The `!` isn't necessary if you add the above `[NotNullWhen(true)]` suggestion
```suggestion
                    if(appearance.SillyColorFilter is not null) {
```